configure.ac

```diff
@@ -439,6 +439,12 @@ AC_LINK_IFELSE([AC_LANG_PROGRAM([[
 
 AC_SUBST(RKT_DEFINES_FOR_ENTER)
 
+# Check whether we can build TPM support code
+AC_MSG_CHECKING([whether C library provides setns function])
```

Comment:
Anyway, is this `AC_MSG_CHECKING` even needed? Doesn't `AC_CHECK_HEADER` print some checking messages?